app/models/owner.rb

```diff
@@ -7,7 +7,7 @@ class Owner < ApplicationRecord
   scope :created_after, ->(date) { where('created_at > ?', date) }
   scope :updated_after, ->(date) { where('updated_at > ?', date) }
 
-  scope :kind, ->(kind) { where(kind: kind) }
+  enum kind: { user: 'user', organization: 'organization' }
```

Comment:
We still need the scope of kind here for the owners controller

Reply:
I reverted the removal of the kind scope